server/NuGet.config

```diff
@@ -3,7 +3,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="VS" value="https://devdiv.pkgs.visualstudio.com/_packaging/VS/nuget/v3/index.json" />
+    <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
```

Comment:
vs-impl is being used due the other workaround due to the signing issues, or is this actually the right feed in this case?

Reply:
This seems like the right feed for the moment.  However it would be harder to change until I fix signing, so if you think there is a better one, I'm happy to change it later.

Reply:
It seems a bit strange that we're using vs-impl since I usually imagine that as a feed we consume that contains packages produced by the VS team where in this case we're wearing our .NET hat, not VS hat. Although maybe we already publish other packages there too?

Reply:
We do indeed, for example all our external access packages.  This seemed to fit with those since its purpose is to be consumed by another one of our components.  Other potential feeds are dotnet-eng or dotnet-tools.  But publishing to any of those needs to wait for my other changes.

Reply:
Discussed offline- we're going to try and switch to the [msft_consumption ](https://dev.azure.com/azure-public/vside/_artifacts/feed/msft_consumption)feed to ensure that the versions we insert here are retained